x-pack/plugins/siem/public/components/navigation/breadcrumbs/index.test.ts

```diff
@@ -34,16 +34,31 @@ describe('Navigation Breadcrumbs', () => {
       const breadcrumbs = getBreadcrumbsForRoute(pathname);
       expect(breadcrumbs).toEqual(rootBreadcrumbs.hosts);
     });
+
     test('should return Host breadcrumbs when supplied host pathname', () => {
       const pathname = `/hosts`;
       const breadcrumbs = getBreadcrumbsForRoute(pathname);
       expect(breadcrumbs).toEqual(rootBreadcrumbs.hosts);
     });
+
     test('should return Network breadcrumbs when supplied network pathname', () => {
       const pathname = `/network`;
       const breadcrumbs = getBreadcrumbsForRoute(pathname);
       expect(breadcrumbs).toEqual(rootBreadcrumbs.network);
     });
+
+    test('should return Timelines breadcrumbs when supplied link-to timelines pathname', () => {
+      const pathname = `/link-to/timelines`;
```

Comment:
These backticks are not needed. I won't comment on any others because eventually we will have a linter rule again to fix all of these. For now, @spong looked it up but it's not available to us as an eslinter rule yet.

Up to you if you want to change 'em.

Reply:
That's the style used throughout the existing code in this file. I'll change all the lines in the existing code where this feedback is also applicable.